### ISSUE TITLE
Suite P theme cannot display left & right menus simultaneously in email

### ISSUE THREAD

**Issue by blloyd78** [salesagility/SuiteCRM#2571]
#### Issue

When using the email compose screen, the Suite P theme does not allow the left and right menu bars to be displayed simultaneously. Clicking the Attach button appears to do nothing until you collapse the left menu bar.

See the screenshots for an example.

#### Expected Behavior

The central pane should scale to allow both left and right menu bars to be displayed simultaneously.

#### Actual Behavior

You must minimise/collapse the left hand menu bar in order to add an attachment to an email

#### Possible Fix

None known

#### Steps to Reproduce

1. Ensure the left-hand menu is not minimised
2. Start to compose a new message
3. Click the Attach button.
4. You will not be able to attach files to the email until you collapse the left-hand menu

#### Context

Because a workaround exists (minimising menu bars) this is probably only a low priority issue

#### Your Environment

* SuiteCRM Version used: 7.7.6
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Safari 10.0.1
* Environment name and version (e.g. MySQL, PHP 7): MySQL PHP 5.6
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 16.04
<img width="1280" alt="screen shot 2016-11-06 at 22 46 19" src="https://cloud.githubusercontent.com/assets/17355746/20042549/e635839c-a473-11e6-8eda-f1b2a45b767f.png">
<img width="1281" alt="screen shot 2016-11-06 at 22 46 00" src="https://cloud.githubusercontent.com/assets/17355746/20042550/e64da440-a473-11e6-9607-3f2ab9606fa6.png">

**Comment by blloyd78** [comment]
Just to confirm that this issue is still present in v7.7.9

**Comment by murdal** [comment]
Added 'On Roadmap' as SuiteCRM 7.9 will introduce a new email client making it more responsive.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Suite P theme cannot display left & right menus simultaneously in email](https://trello.com/c/vn8WXWQu/56-suite-p-theme-cannot-display-left-right-menus-simultaneously-in-email)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
